Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.11.4-alpine
+LABEL maintainer="volodymyr_dolgyi@ukr.net"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+RUN mkdir -p /files/media
+
+RUN adduser \
+    --disabled-password \
+    --no-create-home \
+    my_user

```

Comment:
You can combine the `RUN adduser` command with the `RUN chown` and `RUN chmod` commands to reduce the number of layers in the Docker image. This can be done by using `&&` to chain commands together.